assets/javascripts/redmine_advanced_messenger.js

```diff
@@ -0,0 +1,75 @@
+const PWA_START_URL = "/my/page";
+const PWA_BADGE_VALUE = "PWA_BADGE_VALUE";
+const TIMER_CLOSING_OLD_NOTIFICATIONS = 5; // seconds
+
+// register service worker
+function registerPWAServiceWorker() {
+    if (navigator.serviceWorker) {
```

Comment:
As I see it the `serviceWorker` is register for normal use as well (not only on `PWA`).

Is it possible to register the `serviceWorker` only on PWA? Or is a request to work on Non PWA as well?

Reply:
It is possible to register just for PWA. OK from me to do this change.